ext/web/14_compression.js

```diff
@@ -88,6 +91,9 @@ class DecompressionStream {
         const output = ops.op_compression_finish(rid);
         maybeEnqueue(controller, output);
       },
+      cancel: (_reason) => {
+        ops.op_compression_finish(rid);
```

Comment:
This could be `ops.op_compression_finish(rid, false)`, and we could use `ops.op_compression_finish(rid, true)` for the success case right above.